src/main/interface/cli.c

```diff
@@ -3239,10 +3262,54 @@ static void cliDumpRateProfile(uint8_t rateProfileIndex, uint8_t dumpMask)
     rateProfileIndexToUse = CURRENT_PROFILE_INDEX;
 }
 
+#ifdef USE_CLI_BATCH
+static void cliPrintCommandBatchWarning(const char *warning)
+{
+    cliPrintWarningLinef("Errors were detected - please review before continuing");
+    if (warning) {
+        cliPrintWarningLinef(warning);
+    }
+}
+
+static void resetCommandBatch(void)
+{
+    commandBatchActive = false;
+    commandBatchError = false;
+}
+
+static void cliBatch(char *cmdline)
+{
+    if (strncasecmp(cmdline, "start", 5) == 0) {
```

Comment:
I don't think this is needed at all:
- errors that are made even without the user (or some app by means of hidden magic) explicitly enabling an 'error detect' mode do still matter;
- users can still save, even if there were errors, so there is little value to suppressing error detection for any commands entered in CLI.

If this stays, it should be renamed to `errordetect` or suchlike, since the functionality per se has got nothing to do with batching.